Management/src/main/webapp/WEB-INF/jsp/menu.jsp

```diff
@@ -0,0 +1,7 @@
+
+<%@include file="commons/header.jspf"%>
+<%@include file="commons/navigation.jspf"%>
+<div class="container">
```

Comment:
What is the reason of this empty `div`?

Reply:
This is a part of a template, so I just left this one and remove this line from the JSP fragments where use this template, I don`t know, is this the better approach?